index.js

```diff
@@ -68,3 +69,16 @@ function sortByLast(a, b) {
   }
   return 0
 }
+
+function search() {
+  $("#search").on("keyup", function(event) {
+    var text = $("#search").val().toLowerCase()
+    $(".name").each(function(index) {
+      if ($(this).text().toLowerCase().includes(text)) {
```

Comment:
Yay for `includes()`! This is ES7 syntax and super nice to use. The _only_ thing to keep in mind is that it won't work in all browsers but eventually will. Not a suggestion for change, just a flag.